<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbUnstaged">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Unstaged:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStaget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Staged:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teChanges">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="teCommitMsg">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>501</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btReload">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btAddChanges">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btCommit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Commit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btPush">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Push</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbBranch">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbCommitMsg">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Commit message:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwUnstaged">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>256</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lwStaged">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Repository</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionClone"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="text">
    <string>Clone</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
